pkg/types/default.go

```diff
@@ -17,7 +17,7 @@ const (
 	DefaultAuthDB                      string        = "auth"
 	DefaultLocalAuthCollection         string        = "usercredentials"
 	GithubState                        string        = "github"
-	UserInfoKey                        string        = "userInfo"
+	JWTUserCredentialsKey              string        = "userCredentials"
```

Comment:
2 constants (DefaultLocalAuthCollection and JWTUserCredentialsKey) having the same value?

Reply:
`C` is capital for `JWTUserCredentialsKey`. But not necessary to have 2. We can move them into 1 and name it like `USER_CREDENTIALS = "usercredentials"`

Reply:
Where do we use the other key then?

Reply:
Its the collection name in which the users are stored.